manticore/manticore.py

```diff
@@ -151,7 +151,7 @@ class Manticore(object):
     '''
     The central analysis object.
 
-    :param str binary_path: Path to binary to analyze
+    :param str path_or_state: Path to a binary to analyze or `State` object
```

Comment:
nice - we should also update the `str` type annotation next to `param`. we should use a `:type ...` block to do the multiple types thing